tests/integration_tests/build/test_binary_size.py

```diff
@@ -13,9 +13,9 @@
 
 SIZES_DICT = {
     "x86_64": {
-        "FC_BINARY_SIZE_TARGET": 2067944,
+        "FC_BINARY_SIZE_TARGET": 2172072,
```

Comment:
since this PR did not introduce any code changes I find this increase very weird. do you have any idea why that is?
The only thing that should be changed in the binary is the version of the repo

Reply:
I also found this very strange. I checked and the binary size on the latest commit was already very closed to the maximum allowed.

Reply:
@luminitavoicu how close were we to this limit?

Reply:
About 400 bytes I think

Reply:
[This](https://github.com/firecracker-microvm/firecracker/pull/2696/files#diff-685f53c5da36ac443b59e292b4b176a81f57580025794ca66c2b9fc8de123fcaL3) is the only thing that we're changing in the binary. Strange to see that it increases the binary size so much.

Reply:
Binaries are VERY different it seems
![image](https://user-images.githubusercontent.com/81714/129230542-70fa9184-66ed-4d63-8274-36029fcfca3d.png)